en/faq/annotations/index.md

```diff
@@ -0,0 +1,113 @@
+
+::: tip Markdown support
+
+You can add extra formatting with Markdown by:
+
+- asterisks (\*) around words to make it italic. \*italic text\* will be displayed as *italic text*.
+- double asterisks (\**) around words to make it bold. \*\*bold text\*\* will be displayed as **bold text**.
+- backticks (\`) around a piece of code. \`Variables\` will be displayed as `Variables`.
+
+Here you can view [all the possibilities of Markdown](/en/references/exercise-description/#markdown).
+:::
+
+![img.png](./student.react.png)
+
+You can also respond to an existing question from yourself or to a comment from a teacher. To do this, click on `Reply` below the question or comment. Type your response in the text box and click on `Reply`.
+
+## How do I know if my question has been answered? <Badge type="tip" text="student" />
+
+When a teacher answers your question, you will receive a notifications in Dodona. This notification will redirect you to the page where you asked the question.
+
+![image](./comment_on_code.png)
+
+## How can I enable/disable student questions? <Badge type="tip" text="teacher" />
+
+Use the pencil in the top navigation bar of your course to go to the `edit` page. Here you can enable or disable student questions by checking or unchecking the box `Allow students to ask questions`.
+
+![img.png](./ask_questions_toggle.png)
+
+Don't forget to save your changes by clicking on the pink checkmark at the top right of the page.
+
+## How do I know if there are any open questions? <Badge type="tip" text="teacher" />
+
+If there are open questions within a course, you will see a notification in the course card on your homepage.
+
+![img.png](./course_card_question.png)
+
+You will also see a notification in the top navigation bar of your course. Click on the notification to go to the page where you can see all the open questions.
+
+![img.png](./course_question_list.png)
+
+If you want to see al open question accross all your courses, you can go to the `questions` page in the navigation sidebar of Dodona.
+
+![img.png](./questions_index_page.png)
+
+## How can I reply to a question from a student? <Badge type="tip" text="teacher" />
+
+![Question on code](./question-on-code.png)
+
+You can choose to type a reply to the question by clicking the input field. While typing, the question will be marked as in progress, to notify other teachers someone is already working on it.
+Finishing the reply will mark the question as answered and sends the student a notification.
+
+You can also directly mark a question as answered. This is often useful if you know the student has already solved the question in another way. Eg. the student has submitted an other solution where the problem is fixed, or you have replied to the question outside of the Dodona framework.
+
+> Note: comments support markdown formatting, so you can use bold, italics, lists, ... You can checkout all markdown features in [this reference](/en/references/exercise-description/#markdown).
```

Comment:
Make capital/lowercase spelling of Markdown consistent throughout the text. I suggest "Markdown"